modules/developer_manual/pages/general/backporting.adoc

```diff
@@ -132,6 +146,8 @@ NOTE: This command opens the Pull Request and sets the target branch
 
 === Backporting Alias
 
+You can also create a git alias for backporting making the use even easier.
```

Comment:
I suggest changing this to "You can also create a git alias for backporting, making it simpler to use." Would be good to add a link to creating git aliases as well (https://git-scm.com/book/en/v2/Git-Basics-Git-Aliases),

Reply:
done

Reply:
done